server/gitrest/packages/gitrest-base/src/utils/repositoryManagerBase.ts

```diff
@@ -199,20 +199,15 @@ export abstract class RepositoryManagerBase implements IRepositoryManager {
         api: (...args: T) => Promise<U>,
         apiArgs: T,
         apiName: string,
-        additionalProperties?: Record<string, any>): Promise<U> {
-        const metric = Lumberjack.newLumberMetric(
+        additionalProperties?: Record<string, any>,
+    ): Promise<U> {
+        return executeApiWithMetric(
```

Comment:
Interesting... is there no conflict between this `executeApiWithMetric` (referring to the import) and the private method defined in line 198? At first, I was afraid this could lead to recursive behavior, but I think that would only happen if we had a `this.` keyword preceding `executeApiWithMetric`, right?

Reply:
Correct! `executeApiWithMetric` is distinct from `this.executeApiWithMetric`